### ISSUE TITLE
Please change rating (1-5) shortcuts

### ISSUE THREAD

**Issue by maboleth** [Beep6581/RawTherapee#5603]
It would be far more practical if ratings would use [1-5] keys and 1-5 filter would use [shift+number]. Basically the other way around than currently is.

Why? Because you only have to filter once in a while while assigning 1-5 stars is a job done every time there's a new image. Pressing shift and numbers is awkward and not that fast.

**Comment by Thanatomanic** [comment]
I hardly use ratings at all, but I agree that this would be more logical.

**Comment by maboleth** [comment]
Glad you agree. I use ratings all the time to separate _ho-hum_ and _nailed it_ images. _Junk_ is already built nicely into RT, very intuitive. :-)

**Comment by maboleth** [comment]
Dang, I hoped this change would go live in 5.8, when I read that many shortcuts were changed. :-) 

Keep up the good work guys, capture sharpening is incredible! Thanks.

**Comment by Beep6581** [comment]
This PR proposes that rating be done without using a modifier key. For reference, I checked the two image viewers I have installed. Gwenview supports rating and behaves as proposed here. Geeqie supports "marking" and "rating", where the former does not use a keyboard modifier (and the latter necessarily uses the Alt key). In light of that, and the thumbs-up this issue received, I'll merge it for 5.10.
